### ISSUE TITLE
How to modify the project sampleUffMaskRCNN that caould be used to infer custom datasets?

### ISSUE THREAD

**Issue by MattixLee** [NVIDIA/TensorRT#1154]
Hello,

          Thank you very much for your work.  According to official documents， I installed tensorrt and infer  with mask_rcnn_coco.h5 succeed(sampleUffMaskRCNN).
          The source code comes from matterport/Mask_RCNN, and I modified some parameters in config.py(mask rcnn)，I want to use tensorrt to deploy mask rcnn, which file should I modify? Is there an official tutorial？

         cuda:10.0
         tensorrt 7.0

Thank you very much!

Best wishes!

**Comment by batselem** [comment]
@MattixLee, what version of Keras and tensorflow are you using? I got an error when running mrcnn_to_trt_single.py.

**Comment by MattixLee** [comment]
@batselem Hello, keras 2.1.3   tensorflow 1.14.0,

**Comment by batselem** [comment]
Thank you, I will try with the settings you are using. About your question, you said you succeeded in running sampleUffMaskRCNN on Tensorrt. but you want to modify sampleUffMaskRCNN and run it again on tensorrt? after you modified the mask-rcnn config.py,  have you tried running sampleUffMaskRCNN? IS there any error? If you want to change input size and other parameters in Tensorrt, you may consider modifying mrcnn_config.h.

**Comment by MattixLee** [comment]
@batselem Hi, than you for your reply!

I modified parameters of mask rcnn, and  I want to deploy it with tensorrt. I modified mrcnn_config.h, config.py, mrcnn_to_trt_single.py, but I failed, the infer reslut is none.Coulu you deploy custom mask rcnn with tensorrt sueeceed?
what files should I modify?

Tahnk you very much!

**Comment by batselem** [comment]
@MattixLee, No, I cannot create uff file using mrcnn_to_trt_single.py. In config.py it throws `mrcnn_class_conv1/Reshape/shape` key error which is very strange. These layers should have defined in h5 file.

**Comment by ttyio** [comment]
adding @Tyler-D for visibility.

Hello @MattixLee , here is some good reference https://github.com/NVIDIA/TensorRT/issues/1210#issuecomment-828075248, hope it helps, thanks!

**Comment by nvpohanh** [comment]
Closing for now due to >14 days with no response. Please feel free to reopen if the issue still exists. Thanks
